prow/plugins/config.go

```diff
@@ -621,6 +633,22 @@ func (c *Configuration) TriggerFor(org, repo string) Trigger {
 	return Trigger{}
 }
 
+// DcoFor finds the Dco for a repo, if one exists
+// a Dco can be listed for the repo itself or for the
+// owning organization
+func (c *Configuration) DcoFor(org, repo string) *Dco {
+	if org == "" && repo == "" {
```

Comment:
I don't understand this case, when does this get called with empty `org` and `repo`?

Reply:
Fixed